examples/rsp-next-ts-17/.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
The bulk of this rsp next + react 17 example app is just copy pasta'd from our existing one